src/app/core/rpc/rpx.service.ts

```diff
@@ -1,19 +1,24 @@
 import { Injectable, NgZone } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 
+// multiwallet
+// import rxIpc from 'rx-ipc-electron/lib/renderer';
```

Comment:
Can't do this :P

Reply:
i'm trying everything until I find something that at least works...